sql-select/cheap-to-rent.sql

```diff
@@ -0,0 +1,5 @@
+select "filmId",
+"title"
+from "films"
+order by "rentalRate" asc
+limit 50;
```

Comment:
While this technically works it is a whoops it worked situation. To guarantee you get only moves that cost less than a dollar to rent you want to add a `where` clause here. Something like `where "rentalRate" < 1`